### ISSUE TITLE
add contributors json file

### ISSUE THREAD

**Issue by devvsakib** [devvsakib/hacktoberfest-react-project#58]
we need to strore all our contributors details
Name
Profile Picture
GitHub link
file name: contriutors.json

demo 
```js
{
  "profiles": [
    {
      "name": "DevvSakib",
      "avatarUrl": "https://avatars.githubusercontent.com/u/88339569?v=4",
      "github": "devvsakib",
      "twitter": "devvsakib"
    }
 ]
}
```

for avatar, goto user profile, right click on image, COPY IMAGE LINK. that's it
twitter is optional, you can add your

**Comment by Pragya2056** [comment]
Please assign this to me

**Comment by devvsakib** [comment]
> Please assign this to me

assigned

**Comment by Pragya2056** [comment]
@devvsakib 
Please let me know

**Comment by devvsakib** [comment]
> 



> @devvsakib Please let me know

don't add  twitter now

**Comment by Pragya2056** [comment]
sure

**Comment by Pragya2056** [comment]
I can add twitter of those who have it on. and remove it for those who don't

**Comment by devvsakib** [comment]
> I can add twitter of those who have it on. and remove it for those who don't

exactly!

**Comment by Pragya2056** [comment]
The file was added - #59

**Comment by Pragya2056** [comment]
@devvsakib 
Please review it.

**Comment by devvsakib** [comment]
> @devvsakib Please review it.

really nice job

**Comment by devvsakib** [comment]
Solved by #59
